src/lib/client.ts

```diff
@@ -0,0 +1,11 @@
+import { PostHog } from 'posthog-node'
+
+let _client: PostHog | undefined;
+
+export const getPostHogClient = () => {
+    if (!_client) {
+        _client = new PostHog("sTMFPsFhdP1Ssg", { host: 'https://us.i.posthog.com', flushAt: 1, flushInterval: 0 });
```

Comment:
is this running on the client or server? If it's a server, we can set it as an environment variable so it's easier to rotate if needed.

Reply:
not a blocker